configs/eth-wifi-mbedignore

```diff
@@ -1,3 +1,5 @@
 easy-connect/atmel-rf-driver/*
 easy-connect/mcr20a-rf-driver/*
 easy-connect/stm-spirit1-rf-driver/*
+easy-connect/wifi-x-nucleo-idw01m1/*
```

Comment:
at least esp8266 / k64f seems to compile fine without this. is there some known combination that needs this?